ports/rp2/tusb_port.h

```diff
@@ -0,0 +1,12 @@
+#ifndef TUSB_PORT_H_
+#define TUSB_PORT_H_
+
+enum {
+	VENDOR_REQUEST_WEBUSB = 1,
+	VENDOR_REQUEST_MICROSOFT = 2,
+	VENDOR_REQUEST_WEBUSB_CONNECTION = 34
```

Comment:
Do you have a link for where this vendor request number is from? I can find the other two but seemingly not this one.

Reply:
I got this number from the tinyusb repo in the [webusb example](https://github.com/hathach/tinyusb/blob/aaec12a3fb3e505c00b5abba47a55b1a20f6076f/examples/device/webusb_serial/src/main.c#L202).
This number is then used in the browser code to request to connect and disconnect with the device